cli/lib/exec/spawn.js

```diff
@@ -236,6 +237,21 @@ module.exports = {
         child.on('exit', resolveOn('exit'))
         child.on('error', reject)
 
+        if (isPlatform('win32')) {
+          const rl = readline.createInterface({
+            input: process.stdin,
+            output: process.stdout,
+          })
+
+          // on windows, SIGINT does not propagate to the child process when ctrl+c is pressed
+          // this makes sure all nested processes are closed(ex: firefox inside the server)
+          rl.on('SIGINT', function () {
```

Comment:
Do we need to handle anything else?  `SIGTERM`? `SIGUSR1` or `SIGUSR2`?

Reply:
not that I am aware of but Im not sure on all the scenarios the process could be killed in the terminal. Either way it should be pretty easy to add support for if needed if a regression is detected that we missed